app/models/journable/historic_active_record_relation.rb

```diff
@@ -65,145 +58,108 @@ def initialize(relation, timestamp:)
 
     self.timestamp = Array(timestamp)
     readonly!
-    instance_variable_set :@table, model.journal_class.arel_table
   end
 
-  # We need to patch the `pluck` method of an active-record relation that
-  # queries historic data (i.e. journal data). Otherwise, `pluck(:id)`
-  # would return the `id` of the journal table rather than the `id` of the
-  # journable table, which would be expected from the syntax:
-  #
-  #     WorkPackage.where(assigned_to_id: 123).at_timestamp(1.year.ago).pluck(:id)
-  #
+  # Patch the `pluck` method of an active-record relation
+  # so that columns callers might expect but that do not exist on the journals table are ignored.
   def pluck(*column_names)
     column_names.map! do |column_name|
-      case column_name
-      when :id, "id"
-        "journals.journable_id"
-      when :created_at, "created_at"
-        "journables.created_at"
-      when :updated_at, "updated_at"
-        "journals.updated_at"
+      if model.column_names_missing_in_journal.include?(column_name.to_s)
+        Rails.logger.warn "Cannot pluck column `#{column_name}` because this attribute is not journalized," \
+                          "i.e. it is missing in the #{journal_class.table_name} table."
+        "null as #{column_name}"
```

Comment:
Maybe better to explicitly wrap with `Arel.sql`?
```suggestion
        Arel.sql("null as #{column_name}")
```